spec/calendar.html

```diff
@@ -483,14 +483,41 @@ <h1>ISODaysInMonth ( _year_, _month_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-toisoweekofyear" aoid="ToISOWeekOfYear">
-      <h1>ToISOWeekOfYear ( _year_, _month_, _day_ )</h1>
+    <emu-clause id="sec-temporal-toisoweekofyear" type="abstract operation">
+      <h1>
+        ToISOWeekOfYear (
+          _year_: an integer,
+          _month_: an integer,
+          _day_: an integer,
+        ): an integer
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It implements the logic for the `Temporal.Calendar.prototype.weekOfYear` method, which is used for the ISO 8601 calendar.</dd>
+      </dl>
       <emu-alg>
-        1. Assert: _year_ is an integer.
-        1. Assert: _month_ is an integer.
-        1. Assert: _day_ is an integer.
-        1. Let _date_ be the date given by _year_, _month_, and _day_.
-        1. Return _date_'s week number according to ISO-8601 as an integer.
+        1. Let _dayOfYear_ be ToISODayOfYear(_year_, _month_, _day_).
+        1. Let _dayOfWeek_ be ToISODayOfWeek(_year_, _month_, _day_).
+        1. Let _thursday_ be 4.
+        1. Let _friday_ be 5.
+        1. Let _saturday_ be 6.
+        1. Let _sunday_ be 7.
+        1. Let _dayInAWeek_ be 7.
+        1. If _dayOfWeek_ is 0, then
+          1. Set _dayOfWeek_ to _saturday_.
```

Comment:
this seems confusing to me; it would be clearer to explicitly map the entire 0-6 range to 1-7 mod 7 (also isn't this one supposed to be sunday, not saturday?)